packages/main-ui/public/input.css

```
@@ -1,3 +1,6 @@
 @import "tailwindcss";
 @import "./theme.css";
 
+@layer base {
+  button {}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)